app/assets/stylesheets/application.css

```diff
@@ -10,6 +10,10 @@
  * files in this directory. Styles in this file should be added after the last require_* statement.
  * It is generally better to create a new file per style scope.
  *
- *= require_tree .
```

Comment:
Yeah, I added Max's stuff to govuk.scss.erb (so that we can customise the asset paths).